src/Maestro/SubscriptionActorService/StateModel/ActorStateManager.cs

```diff
@@ -17,21 +15,18 @@ namespace SubscriptionActorService.StateModel;
 /// </summary>
 internal class ActorStateManager<T> where T : class
 {
-    protected const int DefaultDueTimeInMinutes = 5;
+    public const int DefaultDueTimeInMinutes = 5;
```

Comment:
What about moving this constant out of the class?
1. it does not belong to ActorStateManager, but rather to ...Reminder... class.
2. it already public member.